nexus/db-queries/src/db/datastore/organization.rs

```diff
@@ -49,11 +49,27 @@ impl DataStore {
             .authn
             .silo_required()
             .internal_context("creating an Organization")?;
+        self.organization_create_in_silo(opctx, None, organization, authz_silo)
+            .await
+    }
+
+    /// Create a organization in the given silo
+    pub(super) async fn organization_create_in_silo(
```

Comment:
Organization most likely is going away - see RFD 332

Reply:
Right still needed it here but will rebase + remove once those changes land.

Reply:
removed